easybuild/main.py

```diff
@@ -357,6 +373,12 @@ def main(args=None, logfile=None, do_build=None, testing=False, modtool=None):
                     log=_log, opt_parser=eb_go.parser, exit_on_error=not testing)
     _log.debug("Paths: %s", paths)
 
+    # if we are only copying ec's for a specific PR we can do that now
+    if options.copy_ec and options.from_pr:
+        determined_paths = [path[0] for path in paths]
```

Comment:
@ocaisa This looks a bit strange, but the entries in `paths` are basically tuples of `(<path>, <bool>)`, where the `<bool>` indicates whether the path is generated or not.

I know you copy-pasted this from parts below, but this is probably clearer:

```python
determined_paths = [path for (path, _) in paths]
copy_ecs_to_target(determined_paths, target_path)
```

or just

```python
copy_ecs_to_target([path for (path, _) in paths], target_path)
```

or event


```python
copy_ecs_to_target([x for (x, _) in paths], target_path)
```

Please change it below too, to keep things consistent.

Reply:
Hmm, not sure what you mean by doing it below, the case there is very different

Reply:
`regtest([path[0] for path in paths] or ...` -> `regtest([x for (x, _) in paths] or`